no-computer.md

```diff
@@ -7,45 +7,103 @@ good way to meet their needs.
+
+
+_What does programming look like?_
+
+According to Wikipedia, “Braille may be produced by hand using a slate and stylus in which each dot is created from the back of the page, writing in mirror image, or it may be produced on a braille typewriter or Perkins Brailler, or an electronic Brailler or eBrailler.” [https://en.wikipedia.org/wiki/Braille]
+
+_Why is this the right way to interact with the problem domain?_
+
+ This is the right way to interact with the domain because one sense that the visually impaired generally have is their sense of touch. It would be difficult to rely on senses other than one’s sense of touch as a standard form of communication, and of all the senses to be impaired (hearing, smelling), the sense of touch seems to be one that people generally have intact. 
+
 
 
 ### Operation (semantics)
-_What might happen when a program runs? How does a program interact with the
-user? What kinds of errors might occur, and how might they be communicated to
-the user?_
+_What might happen when a program runs?_
+
+Braille isn’t a computer program that is run! :) 
+
+_How does a program interact with the user?_
+
+Braille traditionally interacts with the user by presenting raised dots on a sheet of paper, dots that the user brushes his fingers across to “read” the letters. 
+
+_What kinds of errors might occur, and how might they be communicated to the user?_
+
+1 possible error is that a braille sheet becomes worn down, making it difficult for the user to distinguish between different letters. The user would likely be aware of this upon touching the surface with the worn-down dots. 
 
 
 ### Expressiveness
-_What should be easy to do in this language? What should be possible, but
-difficult? What should be impossible or very difficult?_
+_What should be easy to do in this language?_
+
+Braille not only serves as a replacement for printed texts in many languages, but can also use the concept of raised dots to communicate an image. Given Braille’s origins as a language derived from the Latin alphabet, and given the fact that its letters were organized according to their position within the French alphabet, it should also be easy to represent other phonetic languages in Braille.
+
+_What should be possible, but difficult?_
+
+However, it might be difficult to represent non-phonetic languages in Braille (excluding non-phonetic languages that have romanized representations, such as Chinese), since many non-phonetic languages do not require the reader to read all the components of a word / character in succession from left to right or right to left. 
+
```

Comment:
This is an interesting point that I had not considered before.  This issue probably comes up in other contexts also. For example, I wonder how languages without alphabets are able to communicate by typing, since you wouldn't want to have an individual key for each word.  Finding the solution to this problem might reveal a solution to dealing with Braille in pictographic scripts.